cns/client.go

```diff
@@ -37,6 +37,7 @@ const (
 	ReleaseVSAN67u3 = "vSAN 6.7U3"
 	ReleaseVSAN70   = "7.0"
 	ReleaseVSAN70u1 = "vSAN 7.0U1"
+	ReleaseVSAN70u2 = "vSAN 7.0U2"
```

Comment:
IMHO you need to update the `switch` statement to check against this version too?

Reply:
My bad. I didn't check the usages of the existing constants as I intended to only use the new constant in csi tests. I would need to re-check this change with my team. To also follow the CLA instructions, I'll close this PR and join the vmware org before re-opening. Thanks.